pkg/daemon/ceph/config/info.go

```diff
@@ -63,6 +63,7 @@ func NewMonInfo(name, ip string, port int32) *MonInfo {
 func (c *ClusterInfo) Log(logger *capnslog.PackageLogger) {
 	mons := []string{}
 	for _, m := range c.Monitors {
+		// Sprintf formatting is safe
```

Comment:
I don't think this comment is really relevant, linking https://github.com/rook/rook/issues/4575 would help

Reply:
let me be clear here, you are asking me to link the issue id in the comment.

Reply:
Yes because the comment as it is is not helping, it has no context, nothing.

Reply:
ok understood, thanks for clearing it up.